ui/packages/components/src/FunctionConfiguration/FunctionConfiguration.tsx

```diff
@@ -8,38 +7,46 @@ import ConfigurationTable, {
   type ConfigurationEntry,
 } from '@inngest/components/FunctionConfiguration/ConfigurationTable';
 import { PopoverContent } from '@inngest/components/FunctionConfiguration/FunctionConfigurationInfoPopovers';
-import { Header } from '@inngest/components/Header/Header';
-import { InvokeButton } from '@inngest/components/InvokeButton';
 import { Pill } from '@inngest/components/Pill';
+import { Time } from '@inngest/components/Time';
 import { Tooltip, TooltipContent, TooltipTrigger } from '@inngest/components/Tooltip';
+import { useCron } from '@inngest/components/hooks/useCron';
 import { AppsIcon } from '@inngest/components/icons/sections/Apps';
 import { EventsIcon } from '@inngest/components/icons/sections/Events';
 import { FunctionsIcon } from '@inngest/components/icons/sections/Functions';
-import { RiArrowRightSLine, RiArrowRightUpLine, RiCloseLine, RiTimeLine } from '@remixicon/react';
-import { toast } from 'sonner';
+import { relativeTime } from '@inngest/components/utils/date';
+import { RiArrowRightSLine, RiArrowRightUpLine, RiTimeLine } from '@remixicon/react';
 
+import type { GetFunctionQuery as DashboardGetFunctionQuery } from '../../../../apps/dashboard/src/gql/graphql';
 import {
   FunctionTriggerTypes,
-  useInvokeFunctionMutation,
-  type GetFunctionQuery,
+  type GetFunctionQuery as DevServerGetFunctionQuery,
 } from '../../../../apps/dev-server-ui/src/store/generated';
 
+type InngestFunction =
+  | NonNullable<DevServerGetFunctionQuery['functionBySlug']>
+  | NonNullable<DashboardGetFunctionQuery['workspace']['workflow']>;
```

Comment:
it's a bit weird for this shared component to reach into both dev-server and cloud dashboard to get their graphql types, but even though they are using different GQL queries, this union type worked well enough for type checking.